content/CORETutorials/JailsPluginsVMs/Plugins/MinIOPlugin.md

```diff
@@ -0,0 +1,208 @@
+
+After migrating data, you can either create an archive of the TrueNAS S3 service or delete it.
+
+### Installing the MinIO Client
+
+The [MinIO Client](https://min.io/docs/minio/linux/reference/minio-mc.html) (`mc`) is command line tool with support for both filesystems and S3-compatible cloud storage services.
+Install `mc` to facilitate data the migration.
+Versions are available for Windows, Linux, and Mac.
+
+This tutorial uses the Windows Subsystem for Linux (WSL), but any standard Linux distribution follows the same process.
+Users should be familiar with WSL or the standard Linux CLI interface.
+
+{{< hint type=important >}}
+Installed MinIO client software version must be between **RELEASE.2022-06-26T18-51-48Z** and **RELEASE.2022-10-29T10-09-23Z**.
+Newer or older versions will not work properly.
+{{< /hint >}}
+
+To install the most recent compatible version of `mc`, enter:
+
+```
+curl https://dl.min.io/client/mc/release/linux-amd64/archive/mc.RELEASE.2022-10-29T10-09-23Z \
+  --create-dirs \
+  -o $HOME/minio-binaries/mc
+
+chmod +x $HOME/minio-binaries/mc
+export PATH=$PATH:$HOME/minio-binaries/
+```
+
+{{< trueimage src="images/CORE/13.0/MinioClientInstall.png" alt="MinIO Client Installed" id="MinIO Client Installed" >}}
+
+After installation completes, enter `mc --version` to confirm the compatible `mc` version is installed.
+
+### Configuring the Minio Plugin for Migration
+
+Due to incompatibility between versions, `mc` is unable to export configuration data from the S3 service.
+Manual configuration of the Mino plugin is needed to enable data migration.
+
+1. If you have not done so, configure the network ports for the Minio plugin.
+
+    a. Go to **Jails** and locate the jail matching the **Jail Name** entered during [installation of the Minio plugin](#installing-the-minio-plugin).
+        {{< trueimage src="images/CORE/13.0/MinioPluginJail.png" alt="Jails Screen" id="Jails Screen" >}}
+
+    b. Click <i class="material-icons" aria-hidden="true" title="Expand">chevron_right</i> to expand details and management options.
+
+    c. Click <i class="fa fa-stop" aria-hidden="true" title="Stop"></i>&nbsp;**STOP** to stop the jail before making any changes.
+
+    d. Click <i class="material-icons" aria-hidden="true" title="Edit">edit</i>&nbsp;**EDIT** to open the **Jails / Edit** screen. Then click <i class="material-icons" aria-hidden="true" title="Expand">chevron_right</i>&nbsp;to expand **Network Properties**.
+        {{< trueimage src="images/CORE/13.0/MinioPluginPortForwarding.png" alt="Network Properties NAT Port Forwarding" id="Network Properties NAT Port Forwarding" >}}
+
+    e. Edit the **Host Port Number** values to use any currently unused ports.
+        The Minio plugin must use different ports than the S3 service, which uses 9000 and 9001 by default.
```

Comment:
maybe this as a more definitive instruction. Do we tell them why they should not use the default ports?
```suggestion
       Do not use the MinIO plugin default ports 9000 and 90001. You must use different ports than the S3 service, 
```